selfdrive/manager.py

```diff
@@ -61,6 +61,23 @@ def unblock_stdout():
     exit_status = os.wait()[1] >> 8
     os._exit(exit_status)
 
+def format_spinner_error(err):
+  err_list = [[]]
```

Comment:
This should only be a few lines. Use the built in textwrap module, which will deal with all the edge cases (such as no spaces to break on).

Reply:
Got it, moved it to textwrap. After upping the max length from 50 to 65 the output from textwrap is looking a lot more similar to the function I had. At 50 it was very close to nanovg so I chose to write my own with a little more leniency. But it looks good now.